stable/kubernetes-dashboard/values.yaml

```diff
@@ -113,12 +113,13 @@ ingress:
   #   - kubernetes-dashboard.domain.com
 
   ## Kubernetes Dashboard Ingress TLS configuration
-  ## Secrets must be manually created in the namespace
+  ## Secrets can be manually created in the namespace
+  ## or will be generated automatically.
   ##
   # tls:
-  #   - secretName: kubernetes-dashboard-tls
-  #     hosts:
-  #       - kubernetes-dashboard.domain.com
+  #   secretName: ""
+  #   hosts:
+  #     name: kubernetes-dashboard.domain.com
```

Comment:
This only allows one secret. This could be a problem for some folks.

Reply:
Yep, that's the simplest situation I thought before (Dashboard maybe just need one entrance is enough.)

But I'll try to make it able to add multi hosts.